cms/djangoapps/contentstore/views/course.py

```diff
@@ -505,7 +505,11 @@ def course_listing(request):
     optimization_enabled = GlobalStaff().has_user(request.user) and \
         WaffleSwitchNamespace(name=WAFFLE_NAMESPACE).is_enabled(u'enable_global_staff_optimization')
 
-    org = request.GET.get('org', '') if optimization_enabled else None
+    organizations = get_organizations() if optimization_enabled else []
+    org_names_list = [(org["short_name"]) for org in organizations]
```

Comment:
Add a safe getter for "short_name". Also, do not mix single and double quotes. @elvinquero 
nit: you can remove this variable as it's only used once.